### PR TITLE
fix: fix list difference precedence

### PR DESCRIPTION
**Pull request by thalesmg**
```erlang
1> [a, b, c] ++ [d] -- [a].
[a,b,c,d]
2> ([a, b, c] ++ [d]) -- [a].
[b,c,d]
```

### CODE REVIEW

**Comment by zmstone** [comment]
Question: what does it mean to rest APIs?
Are the apis for dashboard served by management apis too?

**Comment by thalesmg** [comment]
> Question: what does it mean to rest APIs? Are the apis for dashboard served by management apis too?

Apparently, yes: the dashboard API is available on both listeners prior to this fix:

```
ͳ for p in 18083 8081; do echo $p: ; curl -su admin:public http://localhost:$p/api/v4/users | jq .; done
18083:
{
  "data": [
    {
      "username": "admin",
      "tags": "administrator"
    }
  ],
  "code": 0
}
8081:
{
  "data": [
    {
      "username": "admin",
      "tags": "administrator"
    }
  ],
  "code": 0
}
```

After:

```sh
ͳ for p in 18083 8081; do echo $p: ; curl -su admin:public http://localhost:$p/api/v4/users | jq .; done
18083:
{
  "data": [
    {
      "username": "admin",
      "tags": "administrator"
    }
  ],
  "code": 0
}
8081:
parse error: Invalid numeric literal at line 1, column 4
```

**Review by zmstone — DISMISSED** [review]

**Comment by zmstone** [comment]
Forgot one thing. Please update CHANGES

File: CHANGES-4.3.md [hunk 1]
```diff
@@ -34,6 +34,8 @@ File format:
 - Ensure start dashboard ok event if default_username is missing.
 - Fix key update from JWKS server by JWT auth. [#8337]
 - Better errors for JWT claim validations. [#8337]
+- Fixed issue where the dashboard APIs were being exposed under the
```
Comment (zmstone) [inline CHANGES-4.3.md:37]: 4.3.16 is released

Comment (thalesmg) [reply CHANGES-4.3.md]: :see_no_evil:

**Review by zmstone — APPROVED** [review]

**Pull request merged**